packages/useTrie/__tests__/index.test.ts

```diff
@@ -165,17 +161,20 @@ describe('Object array tests', () => {
 
       const trie = new Trie(words1, isCaseInsensitive, o => o.title);
 
-      expect(trie.has('aBc')).toBe(true);
-      expect(trie.has('aBC')).toBe(true);
-      expect(trie.has('ABC')).toBe(true);
-      expect(trie.has('abc')).toBe(true);
-      expect(trie.has('AbC')).toBe(true);
-
-      expect(trie.has('AbD')).toBe(true);
-      expect(trie.has('aBD')).toBe(true);
-      expect(trie.has('ABD')).toBe(true);
-      expect(trie.has('abd')).toBe(true);
-      expect(trie.has('aBd')).toBe(true);
+      [
+        'aBc',
+        'aBC',
+        'ABC',
+        'abc',
+        'AbC',
+        'AbD',
+        'aBD',
+        'ABD',
+        'abd',
+        'aBd',
+      ].forEach(word => {
+        expect(trie.has(word)).toBe(true);
```

Comment:
Just making the tests a little easier to maintain. 😉

Reply:
This is surely a lazier way of maintain :+1:  not to mention more readable